docs/shared/guides/global-nx.md

```diff
@@ -0,0 +1,122 @@
+# Managing your Global Nx Installation
+
+Nx can be ran in a total of 3 ways:
+
+- Through your package manager (e.g. `npx nx`, `yarn nx`, or `pnpm exec nx`)
+- Through [./nx or ./nx.bat](/more-concepts/nx-and-the-wrapper)
+- Through a global Nx installation (e.g. `nx`)
+
+With a global Nx installation, Nx looks for the local copy of Nx in your repo and hands off the process execution to it. This means that whichever version of Nx is installed locally in your repo is still the version of Nx that runs your code. For the most part, this can eliminate any issues that may arise from the global install being outdated.
+
+However, there are still cases where an issue could arise. If the structure of your Nx workspace no longer matches up with what the globally installed copy of Nx expects, it may fail to hand off to your local installation properly and instead error. This commonly results in errors such as:
+
+- Could not find Nx modules in this workspace.
+- The current directory isn't part of an Nx workspace.
+
+If you find yourself in this position, you will need to update your global install of Nx.
+
+## Updating your global Nx installation
```

Comment:
Let's call this `Installing your global Nx installation` and move after the paragraph on line 9.

We can then refer to it for update section and later when we remove the packages.